ocs_ci/ocs/constants.py

```diff
@@ -378,6 +378,8 @@
     TEMPLATE_COUCHBASE_SERVER_DIR, "couchbase-worker-example.yaml"
 )
 
+COUCHBASE_OPERATOR = 'couchbase-operator-namespace'
```

Comment:
It is mandatory to deploy couchbase only in this namespace?
If not then we should give user an option to deploy in his own defined namespace

Reply:
Need to work on it. Thanks, I missed it

Reply:
According to https://docs.couchbase.com/operator/1.1/operator-config.html, a namespace can be changed. I'm trying to understand what's the benefits of having custom namespace

Reply:
This way if we stick to a single fixed namespace, we will not be able to deploy multiple couchbases(each CB on a different namespace).
This can be taken in the next PR